pkgs/by-name/rm/rman/package.nix

```diff
@@ -24,7 +24,10 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
-  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=incompatible-function-pointer-types";
+  env.NIX_CFLAGS_COMPILE = lib.concatStringsSep " " (
+    [ "-Wno-error=incompatible-pointer-types" ]
+    ++ lib.optional stdenv.cc.isClang "-Wno-error=incompatible-function-pointer-types"
+  );
```

Comment:
This should either be an `if`/`else` or just include the broader warning.

Reply:
Done. I checked that it still works with clangStdenv.